### PR TITLE
336 contact redesign

### PR DESCRIPTION
**Pull request by P-Saha**
**Description of changes**

**Type of Change**

- [ ] Bug
- [x] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/contact.tsx [hunk 1]
```diff
@@ -32,63 +32,53 @@ const Ul = styled('ul')`
...
+`;
+
 const FlexRow = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
   margin-bottom: 16px;
 `;
 
+function onChange(value) {
```
Comment (hlminh2000) [inline components/pages/contact.tsx]: this callback should really be encapsulated inside `ContactPage` eh, nothing else should gain access to this directly.

Also probably `onReCAPTCHAchange` to differentiate from handlers for other elements. And please provide typing for `value` :)

File: components/pages/contact.tsx [hunk 2]
```diff
@@ -352,8 +352,13 @@ export default function ContactPage() {
                 </FormControl>
               </Col>
             </Row>
-            <Row>
-              <Col align="start">
+            <Row align="end">
+              <Col>
+                <ReCAPTCHA sitekey="6Lfvf-IUAAAAAGJNRhb_urfYwVVBVjOPiQQJbxuz" onChange={onChange} />
```
Comment (hlminh2000) [inline components/pages/contact.tsx]: let's pull `sitekey` from env so we can configure this for each environment

File: components/pages/contact.tsx (hunk 1 above)
Comment (P-Saha) [reply components/pages/contact.tsx]: Fixed!

File: components/pages/contact.tsx (hunk 2 above)
Comment (P-Saha) [reply components/pages/contact.tsx]: In the ReCAPTCHA settings you can configure multiple environments/domains at once. I have still added this as an environment variable however.

File: .env.schema [hunk 3]
```diff
@@ -18,6 +18,10 @@ EGO_CLIENT_ID=platform
 ######### Docs
 DOCS_URL_ROOT = 'https://docs.icgc-argo.org/'
 
+######### ReCAPTCHA
+SITE_KEY = ''
+SECRET_KEY = ''
```
Comment (hlminh2000) [inline .env.schema]: `SECRET_KEY` is a secret, shouldn't be exposed to client (aka, browser). It's not used here, can we remove?

File: .env.schema [hunk 4]
```diff
@@ -18,6 +18,10 @@ EGO_CLIENT_ID=platform
 ######### Docs
 DOCS_URL_ROOT = 'https://docs.icgc-argo.org/'
 
+######### ReCAPTCHA
+SITE_KEY = ''
```
Comment (hlminh2000) [inline .env.schema]: maybe name this `RECAPCHA_SITE_KEY` so it's clearer what it's for

File: .env.schema (hunk 3 above)
Comment (P-Saha) [reply .env.schema]: 🔥

File: .env.schema (hunk 4 above)
Comment (P-Saha) [reply .env.schema]: Renamed :)

File: global/config.ts [hunk 5]
```diff
@@ -18,6 +18,7 @@ export const getConfig = () => {
       `/api/oauth/login/google?client_id=${publicConfig.EGO_CLIENT_ID || ''}`,
     ),
     DOCS_URL_ROOT: publicConfig.DOCS_URL_ROOT || 'https://docs.icgc-argo.org/docs/',
+    RECAPTCHA_SITE_KEY: publicConfig.SITE_KEY || '6Lfvf-IUAAAAAGJNRhb_urfYwVVBVjOPiQQJbxuz',
```
Comment (hlminh2000) [inline global/config.ts]: oops `publicConfig.RECAPTCHA_SITE_KEY` :)

Comment (hlminh2000) [reply global/config.ts]: also can you give this key a shot? :) `6Lebz-IUAAAAACY7eMyfK4H52Sxy9hv4FYjhFgSR` if working, let's use that as default

Comment (P-Saha) [reply global/config.ts]: This key seems to be working, will make default!

**Pull request merged**